docs/post-upgrade-req-cti-alerts

```diff
@@ -0,0 +1,40 @@
+[[post-upgrade-req-cti-alerts]]
+
+= Migrate detection alerts enriched with threat intelligence
+
+After upgrading to {stack} version 7.15.x from a previous minor release (e.g., 7.14.x, etc.), you need to migrate detection alerts enriched with threat intelligence to ensure threat intelligence properly displays in {elastic-sec}.
+
+To migrate detection alerts:
+
+. <<deactivate-detect-rules, Deactivate all detection rules>> (optional).
+
++
+
+IMPORTANT: You should have already deactivated all of your rules before upgrading your {stack}, so only complete this step if you haven't already.
+
++
+
+. Upgrade Kibana. See {ref}/upgrade.html [Upgrade Kibana] for more information.
+. Visit the Overview or Alerts page in {elastic-sec} to update the `.siem-signals*` index.
```

Comment:
Do we often refer to the index/pattern directly like this? It could be useful info, but it might be simpler to just refer to them more generally: 
```suggestion
. Visit the Overview or Alerts page in {elastic-sec} to update the detection alerts index.
```

Reply:
Historically we've referred to this detection alerts index this way, so I'd advocate to keep it as is for the sake of consistency. That said, I agree that "detection alerts index" is easier to read and understand so I'm open to changing things up if the change improves the readability of our docs. @jmikell821 wdty?

Reply:
Also, @rylnd the HTML view doesn't render until we add this file to our TOC. Apologies for the inconvenience :(